sqlmesh/dbt/manifest.py

```diff
@@ -193,17 +193,30 @@ def _load_tests(self) -> None:
 
     def _load_models_and_seeds(self) -> None:
         for node in self._manifest.nodes.values():
-            if node.resource_type not in ("model", "seed"):
+            if node.resource_type not in ("model", "seed", "snapshot"):
                 continue
 
+            if node.resource_type == "snapshot":
+                config = _config(node)
+                if config.get("strategy", "timestamp") != "timestamp":
```

Comment:
in dbt, is None == timestamp?

Reply:
According to current documentation strategy is required. Not sure if this changed though over time. It is also recommended. So, although I think it should always be there, if some old version didn't require it then it would default to timestamp.